### PR TITLE
Add fuse shell doc

### PR DESCRIPTION
**Pull request by bzheng888**
### What changes are proposed in this pull request?
FIx issue #14776
### Why are the changes needed?
Add fuse shell doc 
### Does this PR introduce any user facing changes?
No

### CODE REVIEW

File: docs/en/api/POSIX-API.md [hunk 1]
```diff
@@ -607,3 +607,40 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+
+## Fuse Shell Tool
+The Alluxio JNI-Fuse client provides a useful shell tool to perform some internal operations, such as clearing the client metadata cache. 
+If our Alluxio-Fuse mount point is `/mnt/alluxio-fuse`, the command patten of Fuse Shell is:
+```
+ls -l /mnt/alluxio-fuse/.alluxiocli.[COMMAND].[SUBCOMMAND]
+```
+Among them, the `/.alluxiocli` is the identification string of Fuse Shell, `COMMAND` is the main command (such as `metadatacache`), and `SUBCOMMAND` is the subcommand (such as `drop, size, dropAll`).
+Currently, Fuse Shell only supports `metadatacache` command to clear cache or get cache size, and we will expand more commands and interactive methods in the future. 
+It should be noted that to use the Fuse Shell tool, the configuration item `alluxio.fuse.special.command.enabled` needs to be set to true.
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:619]: To use the Fuse shell tool, `alluxio.fuse.special.command.enabled` needs to be set to true in `${ALLUXIO_HOME}/conf/alluxio-site.properties` before launching the Fuse applications:

File: docs/en/api/POSIX-API.md [hunk 2]
```diff
@@ -607,3 +607,40 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
 - `-d` define the duration. Try to cover the whole POSIX API testing duration
 - `-e` define the profiling target
 - `-f` define the file name to dump the profile information to
+
+## Fuse Shell Tool
+The Alluxio JNI-Fuse client provides a useful shell tool to perform some internal operations, such as clearing the client metadata cache. 
+If our Alluxio-Fuse mount point is `/mnt/alluxio-fuse`, the command patten of Fuse Shell is:
+```
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:614]: Add `console` after the three dot
and add a $ sign before the ls command
$ format
see other places in the doc for example

File: docs/en/api/POSIX-API.md [hunk 3]
```diff
@@ -607,3 +607,40 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+ls -l /mnt/alluxio-fuse/.alluxiocli.[COMMAND].[SUBCOMMAND]
+```
+Among them, the `/.alluxiocli` is the identification string of Fuse Shell, `COMMAND` is the main command (such as `metadatacache`), and `SUBCOMMAND` is the subcommand (such as `drop, size, dropAll`).
+Currently, Fuse Shell only supports `metadatacache` command to clear cache or get cache size, and we will expand more commands and interactive methods in the future. 
+It should be noted that to use the Fuse Shell tool, the configuration item `alluxio.fuse.special.command.enabled` needs to be set to true.
+```
+alluxio.fuse.special.command.enabled=true
+```
+### Metadatacache Command
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:623]: Add empty line before this line

File: docs/en/api/POSIX-API.md [hunk 4]
```diff
@@ -607,3 +607,40 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+When we use Alluxio fuse, we usually turn on the client metadata cache. For example, in a scenario that reads a large number of small files such as AI, enabling client metadata caching can relieve Alluxio Master's metadata pressure and improve read performance.
+When the data in Alluxio is updated, the metadata cache of the client needs to be updated. Usually, you need to wait for the timeout configured by `alluxio.user.metadata.cache.expiration.time` to invalidate the metadata cache.
+This means that there is a time window metadata that may be invalid. In this case, it is recommended to use the metadatacache command of Fuse Shell to clean up the client metadata cache in time. The format of `metadatacache` command is：
+```
+ls -l /mnt/alluxio-fuse/.alluxiocli.metadatacache.[dropAll|drop|size]
+```
+
+####**Usage Example**
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:631]: This line can be removed

File: docs/en/api/POSIX-API.md [hunk 5]
```diff
@@ -607,3 +607,40 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+Currently, Fuse Shell only supports `metadatacache` command to clear cache or get cache size, and we will expand more commands and interactive methods in the future. 
+It should be noted that to use the Fuse Shell tool, the configuration item `alluxio.fuse.special.command.enabled` needs to be set to true.
+```
+alluxio.fuse.special.command.enabled=true
+```
+### Metadatacache Command
+When we use Alluxio fuse, we usually turn on the client metadata cache. For example, in a scenario that reads a large number of small files such as AI, enabling client metadata caching can relieve Alluxio Master's metadata pressure and improve read performance.
+When the data in Alluxio is updated, the metadata cache of the client needs to be updated. Usually, you need to wait for the timeout configured by `alluxio.user.metadata.cache.expiration.time` to invalidate the metadata cache.
+This means that there is a time window metadata that may be invalid. In this case, it is recommended to use the metadatacache command of Fuse Shell to clean up the client metadata cache in time. The format of `metadatacache` command is：
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:626]: add backstick signs around `metadatacache` to show that it's a code than word

File: docs/en/api/POSIX-API.md [hunk 6]
```diff
@@ -607,3 +607,40 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+
+- Clean up all metadata caches：
+```
+ls -l /mnt/alluxio-fuse/.alluxiocli.metadatacache.dropAll
+```
+- Clear the cache of a path and its parent directory under the mount point：
+```
+ls -l /mnt/alluxio-fuse/dir/dir1/.alluxiocli.metadatacache.drop
+```
+The above command will clear the metadata of `/mnt/alluxio-fuse/dir/dir1` and the metadata cache of `/mnt/alluxio-fuse/dir/`.
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:641]: I remember we clean the metadata cache of all the parents, can you help double check?

File: docs/en/api/POSIX-API.md [hunk 7]
```diff
@@ -607,3 +607,40 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+
+- Get the client metadata size
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:643]: Give an example of the output? how the users will know the size?

**Review by LuQQiu — COMMENTED** [review]
Thanks for adding the Fuse shell doc, left some comments

**Review by maobaolong — CHANGES_REQUESTED** [review]
@bzheng888 Thanks for this document, it will helps users to use this great feature quickly, left some comments about the style and grammar, it looks good for the contents.

**Comment by bzheng888** [comment]
@maobaolong @LuQQiu Fix some doc problems, PTAL!

File: docs/en/api/POSIX-API.md [hunk 8]
```diff
@@ -607,3 +607,46 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+## Fuse Shell Tool
+
+The Alluxio JNI-Fuse client provides a useful shell tool to perform some internal operations, such as clearing the client metadata cache. 
+If our Alluxio-Fuse mount point is `/mnt/alluxio-fuse`, the command patten of Fuse Shell is:
+```console
+$ ls -l /mnt/alluxio-fuse/.alluxiocli.[COMMAND].[SUBCOMMAND]
+```
+Among them, the `/.alluxiocli` is the identification string of Fuse Shell, `COMMAND` is the main command (such as `metadatacache`), and `SUBCOMMAND` is the subcommand (such as `drop, size, dropAll`).
+Currently, Fuse Shell only supports `metadatacache` command to clear cache or get cache size, and we will expand more commands and interactive methods in the future.
+To use the Fuse shell tool, alluxio.fuse.special.command.enabled needs to be set to true in ${ALLUXIO_HOME}/conf/alluxio-site.properties before launching the Fuse applications:
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:620]: alluxio.fuse.special.command.enabled surrounded by backstick
to show it's code

Comment (LuQQiu) [reply docs/en/api/POSIX-API.md:620]: Same for ${ALLUXIO_HOME}/conf/alluxio-site.properties

File: docs/en/api/POSIX-API.md [hunk 9]
```diff
@@ -607,3 +607,46 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+```console
+$ alluxio.fuse.special.command.enabled=true
+```
+
+### Metadatacache Command
+
+When we use Alluxio fuse, we usually turn on the client metadata cache. For example, in a scenario that reads a large number of small files such as AI, enabling client metadata caching can relieve Alluxio Master's metadata pressure and improve read performance.
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:627]: When we use Alluxio fuse, we usually turn on the client metadata cache. 
-> 
Client-side metadata cache can be enabled by setting `....=true` to reduce the latency of metadata cache operations and improve FUSE performance in many workloads.

Comment (bzheng888) [reply docs/en/api/POSIX-API.md]: Done!

File: docs/en/api/POSIX-API.md (hunk 8 above)
Comment (bzheng888) [reply docs/en/api/POSIX-API.md]: Done!

**Comment by bzheng888** [comment]
@LuQQiu @maobaolong PTAL!

File: docs/en/api/POSIX-API.md [hunk 10]
```diff
@@ -607,3 +607,47 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
...
+```console
+$ alluxio.fuse.special.command.enabled=true
+```
+
+### Metadatacache Command
+
+Client-side metadata cache can be enabled by setting `alluxio.user.metadata.cache.enabled=true` to reduce the latency of metadata cache operations and improve FUSE performance in many workloads. 
+For example, in a scenario that reads a large number of small files such as AI, enabling client metadata caching can relieve Alluxio Master's metadata pressure and improve read performance.
+When the data in Alluxio is updated, the metadata cache of the client needs to be updated. Usually, you need to wait for the timeout configured by `alluxio.user.metadata.cache.expiration.time` to invalidate the metadata cache.
+This means that there is a time window metadata that may be invalid. In this case, it is recommended to use the `metadatacache` command of Fuse Shell to clean up the client metadata cache in time. The format of `metadatacache` command is：
```
Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:630]: metadata that may be invalid
->
that the cached metadata is outdated

Comment (LuQQiu) [inline docs/en/api/POSIX-API.md:630]: of Fuse Shell to clean up the client metadata cache in time
->
of Fuse Shell to manually clean up the client metadata cache

**Review by LuQQiu — APPROVED** [review]
LGTM, left two minor comments, thanks for adding the fuse shell doc! It's very useful!

File: docs/en/api/POSIX-API.md (hunk 10 above)
Comment (bzheng888) [reply docs/en/api/POSIX-API.md]: Done!

**Comment by bzheng888** [comment]
@LuQQiu PTAL!

**Review by maobaolong — APPROVED** [review]
LGTM

**Comment by bzheng888** [comment]
Ping @LuQQiu

**Pull request merged**